src/github.com/getlantern/lantern/feed.go

```diff
@@ -157,6 +162,7 @@ func GetFeed(locale string, allStr string, proxyAddr string,
 		return
 	}
 
+	ioutil.WriteFile("/tmp/"+locale, contents, 0644)
```

Comment:
If we wanted to write the feed contents to a temporary file on Android, we would want to store it in the ``getCacheDir()`` directory instead of /tmp. If this is just for testing purposes, do we want to only call this if ``runtime.GOOS != 'android'``?